recipe/meta.yaml

```diff
@@ -31,14 +30,15 @@ requirements:
   host:
     - libcurl  # [not win]
     - bzip2  # [not win]
-    # add back when we re-unvendor zlib
-    # - zlib  # [not win]
+    - zlib  # [not win]
 
 test:
   commands:
     - test -f ${PREFIX}/lib/libcfitsio{{ SHLIB_EXT }}  # [not win]
     - if not exist %LIBRARY_LIB%\cfitsio.lib exit 1  # [win]
     - if not exist %LIBRARY_BIN%\cfitsio.dll exit 1  # [win]
+  downstreams:
+    - gdal  # [osx]
```

Comment:
Can you keep this for linux and win as well? The tests don't take that long and it helps keep gdal from failing which seems to be a common occurence. cc @ocefpaf.

Reply:
We need to exclude ppc64le since this env does not solve. Otherwise it should be fine.

Reply:
Ah, yeah. Seems to be a conda-build problem, https://github.com/conda/conda-build/issues/3908

Reply:
```suggestion
    - gdal  # [x86_64]
```